src/components/__counter/Counter.tsx

```diff
@@ -33,7 +45,7 @@ export const Counter = (props) => {
         initialValue = 0,
         step = 1,
         limit = 10,
-        func = (a) => console.log("it works: " + a),
+        func = () => console.log("it works"),
```

Comment:
`func = () => new Error("Counter is used without handler!"),`

Something like that?

Reply:
Just remove them?

Reply:
Yep, better if it breaks during compilation if you don't provide value for this argument - otherwise typescript considers it optional and doesn't warn you.

Reply:
Done